WalletWasabi.Fluent/ViewModels/Wallets/WalletPageViewModel .cs

```diff
@@ -0,0 +1,93 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using ReactiveUI;
+using WalletWasabi.Fluent.ViewModels.Login;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets;
+
+public partial class WalletPageViewModel : ViewModelBase, IEquatable<WalletPageViewModel>, IComparable<WalletPageViewModel>
```

Comment:
I wouldn't implement comparison and equality interfaces because according to [lines 45-47 in `WalletManagerViewModel`](https://github.com/zkSNACKs/WalletWasabi/pull/10459/files#diff-150bbda17bbcc6306836f362ce9c32d015a62c725fc64b720df187d8d31a1455R45-R47) they should not be used. Even if it's used somewhere, I would follow the approach in the mentioned lines since it takes much less code.

Reply:
I did remove all those interfaces from here in #10576, so I'll close resolve your suggestion here, since fixing it in this PR is only going to cause merge conflict headaches.